### PR TITLE
Add gems for running on Windows

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Let me know if this is something you're interested in merging.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -25,7 +25,10 @@ gem 'uglifier', '>= 1.3.0'
 gem 'premailer-rails'
 
 gem 'commonmarker', require: false
-gem 'rugged'
+gem 'rugged' if !Gem.win_platform?
+gem 'rugged', '<= 0.27' if Gem.win_platform?
+gem 'tzinfo-data' if Gem.win_platform?
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```
Comment (iHiD) [inline Gemfile:31]:
```suggestion
platforms: [:mingw, :mswin, :x64_mingw] do
  gem 'rugged', '<= 0.27'
  gem 'tzinfo-data'
  gem 'wdm', '>= 0.1.0'
end

platforms: [:ruby] do
  gem 'rugged'
end
```

Comment (ErikSchierboom) [reply Gemfile]: Much nicer indeed!

**Comment by iHiD** [comment]
This is red.

**Comment by ErikSchierboom** [comment]
@iHiD I'll work on it some more next week

File: Gemfile [hunk 2]
```diff
@@ -25,7 +25,15 @@ gem 'uglifier', '>= 1.3.0'
 gem 'premailer-rails'
 
 gem 'commonmarker', require: false
-gem 'rugged'
+platforms: [:mingw, :mswin, :x64_mingw] do
```
Comment (iHiD) [inline Gemfile:28]:
```suggestion
platforms :mingw, :mswin, :x64_mingw do
```

File: Gemfile [hunk 3]
```diff
@@ -25,7 +25,15 @@ gem 'uglifier', '>= 1.3.0'
...
 
 gem 'commonmarker', require: false
-gem 'rugged'
+platforms :mingw, :mswin, :x64_mingw do
+  gem 'rugged', '<= 0.27'
+  gem 'tzinfo-data'
+  gem 'wdm', '>= 0.1.0'
+end
+
+platforms: [:ruby] do
```
Comment (iHiD) [inline Gemfile:34]:
```suggestion
platforms :ruby do
```

File: Gemfile [hunk 4]
```diff
@@ -25,7 +25,15 @@ gem 'uglifier', '>= 1.3.0'
 gem 'premailer-rails'
 
 gem 'commonmarker', require: false
-gem 'rugged'
+platforms :mingw, :mswin, :x64_mingw do
+  gem 'rugged', '<= 0.27'
```
Comment (SleeplessByte) [inline Gemfile:29]: FYI: rugged 1.0.1 works on Windows (proof: me installing it today)

**Comment by ErikSchierboom** [comment]
I think we can close this, as we'll focus our efforts on v3 instead and have a nice Docker setup there.

**Pull request closed without merging**